api.json

```diff
@@ -1201,15 +1347,15 @@
         "REJECTED",
         "FAILED"
       ],
-      "description": ""
+      "description": "Indicates a refund\u0027s current status."
```

Comment:
All of these descriptions are not swagger compliant, you can see all the warnings if you try pasting into the swagger editor.